lizrd/scripts/grid.py

```diff
@@ -97,21 +100,24 @@
             print("Aborting...")
             exit(1)
 
-    if not (interactive_debug_session or CLUSTER_NAME == MachineBackend.LOCAL):
+    if CLUSTER_NAME != MachineBackend.LOCAL:
         first_exp_training_args, _ = grid[0]
         exp_name = first_exp_training_args["name"]
         name_for_branch = (
             f"{exp_name}_{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}"
         )
         copy_and_version_code(name_for_branch, name_for_branch, False)
     else:
-        print(
-            f"Running in debug mode or locally, skip copying code to a new directory."
-        )
+        print(f"Running locally, skip copying code to a new directory.")
 
     slurm_command = "srun" if interactive_debug_session else "sbatch"
 
     for i, (training_args, setup_args) in enumerate(grid):
+        full_config_path = f"full_config{i}.yaml"
+        with open(full_config_path, "w") as f:
+            yaml.dump({**training_args, **setup_args}, f)
+        training_args["all_config_paths"] += f",{full_config_path}"
+
```

Comment:
maybe there should directory in run_configs where we save there under new directory  and add to .gitignore

Reply:
it's remote, so cannot be done. It will all be in neptune, and the original config is locally present.